CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Changelog
+- Added `assert_items_exclude`.
```

Comment:
The changelog entry should be appended to the end of the "Unrelased" section.

Reply:
Thanks, fixed.